WORKSPACE

```diff
@@ -14,7 +14,7 @@
 workspace(name = "bazel_toolchains")
 
 load(
-    "//skylib:package_names.bzl",
+    "//rules/skylib:package_names.bzl",
```

Comment:
@bazel_toolchains

Reply:
WORKSPACE files cannot be referenced from another repo. We're adding @bazel_toolchains only to targets that we expect will be used from outside this repo (mainly the docker autoconfig rule, container creation rules and container layers/builders) so that it's possible to do so w/o issues.

Reply:
Sorry... didn't realize it's the WORKSPACE file.